docs/import/create_disaster_lib.js

```diff
@@ -1,44 +1,73 @@
 export {createDisasterData};
-export {incomeKey, snapKey, sviKey, totalKey};
+export {BUILDING_COUNT_KEY, incomeKey, snapKey, sviKey, totalKey};
 // For testing
-export {assetDataTemplate};
+export {deepCopy, flexibleAssetData, stateAssetDataTemplate};
 
 const snapKey = 'HD01_VD02';
 const totalKey = 'HD01_VD01';
 const sviKey = 'RPL_THEMES';
 const incomeKey = 'HD01_VD01';
+const BUILDING_COUNT_KEY = 'BUILDING COUNT';
 
-// Has all the basic fields needed for the score asset to be created: SNAP, SVI,
-// and income together with the columns of each, and optional damage asset path.
-// The default values for column names here are taken from Census American
-// FactFinder and CDC tables.
+const commonAssetDataTemplate = Object.freeze({
+  damageAssetPath: null,
+  damageLevelsKey: null,
+  noDamageValue: null,
+});
+
+const stateBasedDataTemplate = Object.freeze({
+  scoreBoundsCoordinates: null,
```

Comment:
This PR seems like a good time to do a documentation of the firestore keys in a README somewhere

Reply:
As you know, I'm a fan of keeping docs and code together when possible. I can add inline comments to the fields in this file, would you like that? It would help to understand who the intended audience is. My impression is that someone who needs to know what a Firestore field means is the same as someone who needs to know what some global variable in our code means, and they should be documented the same way. Do you envision a different audience?

Reply:
I'm good with that! All the keys are in this file right?

Reply:
Done. The `layers` keys aren't in here, neither are mapBounds, etc., but that can be for another day.